feat: number shortening in German

barredterra opened the pull request:
Make sure #17520 works for exclusively German-speaking countries (sorry, Switzerland).

> no-docs

barredterra committed:
feat: number shortening in German

barredterra committed:
feat: don't forget Liechtenstein

shariquerik commented on frappe/public/js/frappe/utils/utils.js:
```diff
@@ -1136,7 +1136,23 @@ Object.assign(frappe.utils, {
 	},
 
 	get_number_system: function (country) {
+		const german_speaking = [
```
```suggestion
		const german_number_system = [
```

ankush commented:
> (sorry, Switzerland)

Let's see if this is doable with language-specific translation. Same number system different shorthand using context-specific translations 😄 


re: https://github.com/frappe/frappe/pull/17520#issuecomment-1199503230

barredterra committed:
feat: number system

- extract into a separate file
- add other countries with indian system
- add german translations

barredterra commented:
@ankush unfortunately, context specific translations seem to be broken at the moment. But apart from that, this should work.

German-speaking countries can just use the default number system, with translated symbol. Similar to the Indian system, which is also used in other countries and languages.

barredterra committed:
refactor: get_number_system
